Assets/Shopify/UIToolkit/Shops/Debug/DebugSingleProductShop.cs

```diff
@@ -40,8 +41,16 @@ public class DebugSingleProductShop : MonoBehaviour, ISingleProductShop {
             LogEvent("OnProductLoaded", product, variants);
         }
 
-        public void OnCartQuantityChanged(int newQuantity) {
-            LogEvent("OnCartQuantityChanged", newQuantity);
+        void IShop.OnCartQuantityChanged(int totalNumberOfCartItems) {
```

Comment:
nit: should this be long as well?

Reply:
I don't think so since this isn't a value driven by the GraphQL API. `int` should be alright here.